geode/dispatcher.py

```diff
@@ -205,13 +206,16 @@ def __init__(self, config=None):
 
         return res
 
-    async def distance_pairs(self, origins, destinations, max_meters=MAX_METERS, sem=None, session=None, provider=None):
+    async def distance_pairs(self, origins, destinations, max_meters=MAX_METERS, sem=None, session=None, provider=None, return_inverse=False):
```

Comment:
What should I reasonably expect `return_inverse` to do here? Can we add docstrings?

Reply:
sure